src/github-release.ts

```diff
@@ -75,26 +111,43 @@ export class GitHubRelease {
     // Remove 'autorelease: pending' which indicates a GitHub release
     // has not yet been created.
     await this.gh.removeLabels(this.releasePR.labels, candidate.pullNumber);
+    return this.releaseResponse({
+      release,
+      version: candidate.version,
+      sha: candidate.sha,
+      number: candidate.pullNumber,
+    });
+  }
 
-    const parsedVersion = parse(candidate.version, {loose: true});
+  releaseResponse(options: {
```

Comment:
nit: I might call this `params` rather than `options`, as `options` feels like you're configuring something, `params` feels like you're setting parameters for the response.